src/_live-monitor.ts

```
@@ -12,6 +12,7 @@ import type { LoadedUsageEntry, SessionBlock } from './_session-blocks.ts';
 import type { CostMode, SortOrder } from './_types.ts';
 import { readFile } from 'node:fs/promises';
 import path from 'node:path';
+import { Result } from '@praha/byethrow';
```

ESLint

[error] 'Result' is defined but never used.

(unused-imports/no-unused-imports)